README.md

```diff
@@ -69,59 +72,73 @@
+### IMPORTANT NOTICE
 
-### `pip3`
+**THIS REPOSITORY DEFAULT BRANCH CONTAINS A DEVELOPMENT VERSION OF `PYANI` (v0.3+). IT HAS A DIFFERENT COMMAND-LINE INTERFACE THAN THE STABLE `PYANI` VERSION (v0.2.x).**
 
-```bash
-pip3 install pyani
-```
+### Documentation for stable version `pyani` v0.2.x
+
+- [`pyani` v0.2.x Documentation](./README_v_0_2_x.md)
+
+**IF YOU WISH TO INSTALL THE STABLE VERSION OF `PYANI`, PLEASE FOLLOW THESE INSTRUCTIONS FOR INSTALLING v0.2.x**
+
+The easiest way to install `pyani` v0.2 is to use `conda` or  `pip`. `conda` is recommended for the simplest installation of third-party tool dependencies (`mummer` and `BLAST`/`BLAST+`).
+
+### `conda`
 
-From version 0.1.3.2 onwards, this should also install all the required Python package dependencies. Prior to this version (i.e. 0.1.3.1 and earlier), you can acquire these dependencies with `pip -r`, and pointing at `requirements.txt` from this repository:
+You will need to install the `bioconda` channel, following instructions at [https://bioconda.github.io/user/install.html](https://bioconda.github.io/user/install.html). Then, to create a new environment for `pyani` and install the program, issue the following command:
 
 ```bash
-pip3 install -r requirements.txt
-pip3 install -r requirements-pip.txt
+conda create --name pyani_env python=3.8 -y
+conda activate pyani_env
+conda install pyani
 ```
 
-### `bioconda`
+### `pip`
 
-With a working `anaconda` installation, install the `bioconda` and `conda-forge` channels:
+`pip` will install `pyani` and its Python dependencies, but not the third-party tools.
 
 ```bash
-conda config --add channels defaults
-conda config --add channels bioconda
-conda config --add channels conda-forge
+pip3 install pyani
 ```
 
-Then install `pyani`:
+### Third-party tools
+
+Three alignment packages are required, to use all of `pyani`'s methods: `mummer`, `BLAST+`, and legacy `BLAST`.
+
+The simplest route to obtaining these tools is to use `conda`/`bioconda`:
 
 ```bash
-conda install pyani
+conda install mummer blast legacy-blast -y
 ```
 
-**NOTE:** The current `conda` version is *behind* the development version and has a different command-line interface. Documentation for the `conda` version (0.2.x) can be found at the link below:
-
-- [https://github.com/widdowquinn/pyani/blob/master/README_v_0_2_x.md](https://github.com/widdowquinn/pyani/blob/master/README_v_0_2_x.md)
+But they can also be installed by following instructions from the tools' own websites.
 
+- **BLAST+** (for `anib`) [ftp://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/LATEST/](ftp://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/LATEST/)
+- **legacy BLAST** (for `aniblastall`) [ftp://ftp.ncbi.nlm.nih.gov/blast/executables/release/LATEST/](ftp://ftp.ncbi.nlm.nih.gov/blast/executables/release/LATEST/)
+- **MUMmer** (for `anim`) [http://mummer.sourceforge.net/](ftp://ftp.ncbi.nlm.nih.gov/blast/executables/release/LATEST/)
```

Comment:
In the [preview](https://github.com/widdowquinn/pyani/blob/efc1155e0668ae6ac9d204f0cd9c84f47070657f/README.md), it looks like none of these links are actually being created. I don't know why this would be as the link for the bioconda site earlier works and they seem to be formatted correctly.

Reply:
The *in situ* GitHub Markdown rendering is a bit of a mystery to me. It's not always consistent. VS Code renders the links correctly, and I would expect Jekyll/GitHub pages to render correctly, also.

If the links are visible, they can be copy/pasted, so it's not a fatal issue. Worth keeping an eye on, though.